product_docs/docs/migration_toolkit/55/mtk_rel_notes/mtk_5580_rel_notes.mdx

```diff
@@ -0,0 +1,17 @@
+| Feature      | Added connection retry capabilities to avoid the failure of data migration operations when the connection to the target database is lost.<br /><br />Use the `connRetryCount`, `connRetryInterval`, and `abortConnOnFailure` options with `-dataOnly` to configure the connection retry operation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| Enhancement  | Added support for the migration of `procedure` object types when migrating from one Postgres server to another.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| Enhancement  | Added new command line options to let you exclude object types during the migration.<br /><br />The `-excludeSequences` option skips a subset of sequences when all sequences are selected for migration.<br /><br />The `-excludeProcs` option skips a subset of procedures when all procedures are selected for migration.<br /><br /> The `-excludeFuncs` option skips a subset of functions when all functions are selected for migration.<br /><br />The `-excludePackages` option skips a subset of packages when all packages are selected for migration.<br /><br />The `-excludeQueues` option skips a subset of queues when all queues are selected for migration.<br /><br />The `-excludeSynonyms` option skips a subset of sequences when all sequences are selected for migration. |
```

Comment:
REPLACE 
"The `-excludeSynonyms` option skips a subset of sequences when all sequences are selected for migration."
WITH
"The `-excludeSynonyms` option skips a subset of synonyms when all synonyms are selected for migration."